docs2/support_matrix.qmd

```diff
@@ -32,3 +32,56 @@ The changes will show up in the dev docs when your PR is merged!
 ## Raw Data
 
 You can also download data from the above tables in [CSV format](./raw_support_matrix.csv).
+
+The code used to generate the linked CSV file is below.
+
+```{python}
+#| code-fold: true
```

Comment:
I'm going to revert this until the next follow-up, since in that follow-up we can reuse `gen_matrix.py` directly by invoking it with `!python ...`.